client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -194,4 +194,12 @@ export const siteSetupFlow: Flow = {
 
 		return { goNext, goBack, goToStep, submit };
 	},
+
+	useAssertConditions() {
+		const siteSlug = useSiteSlugParam();
+
+		if ( ! siteSlug ) {
```

Comment:
We will need to support (`siteId` OR `siteSlug`) in the future because when a user purchases a domain in signup, they actually don't have a siteSlug yet and so `siteId` is passed (at least that's how it works in the `/start` flow, @alshakero might know if this flow handles that case in another way?) p1649730710637599-slack-C02T4NVL4JJ

Reply:
Thanks @roo2, in my last commit I should have addressed the `siteId` support.